### ISSUE TITLE
Discord webhooks not working [1.20.1]

### ISSUE THREAD

**Issue by LazyDope** [Patbox/BanHammer#39]
Setup config.json with the webhook url list like so
```
"discordWebhookUrls": [
    "https://discord.com/api/webhooks/<id>/<webhook_token>"
  ]
```
and it worked for a few weeks, but suddenly stopped working, with no clear indication from the logs that the messages are failing to send. Possibly an issue with the Discord API?

**Comment by arnokeesman** [comment]
Yeah discord removed an old way of doing things, it's been fixed by #38 
Though that might not have been released for your Minecraft version

**Comment by LazyDope** [comment]
Yes we are still using 1.20.1 because we're waiting for other mods to update, and it seems like the mod is only released for the latest version of Minecraft.
